--- fantastik-2.0/fantastik.theme.old	2005-03-01 17:32:34.299994440 +0100
+++ fantastik-2.0/fantastik.theme	2005-03-01 17:35:39.946771840 +0100
@@ -43,8 +43,8 @@
 text x=192 y=148 sensor=network device="ppp0" format="%out kB/s" align=right color=255,0,0 fontsize=9 font="Tahoma" interval=250
 
 # Mail
-clickarea x=15 y=170 w=32 h=32 onclick="thunderbird"
-text x=55 y=178 sensor=program program="/home/aivs/fantastik-2.0/programs/mails_pop3.pl pop.yandex.ru login password" line=1 color=30,30,30 fontsize=10 font="Tahoma" interval=18000
+clickarea x=15 y=170 w=32 h=32 onclick="kmail"
+text x=55 y=178 sensor=program program="/usr/share/themes/superkaramba/fantastik/programs/mails_pop3.pl pop.server.com login password" line=1 color=30,30,30 fontsize=10 font="Tahoma" interval=18000
  
 #Time
 text x=90 y=213 sensor=time format="dddd" align=center color=30,30,30 fontsize=10 font="Tahoma"
